GenderPayGap.WebUI/Views/ManageOrganisations/AllOrganisationReports.cshtml

```diff
@@ -0,0 +1,329 @@
+            var newRow = new TableRowViewModel
+            {
+                Row = new List<TableCellViewModel>
+                {
+                    new TableCellViewModel
+                    {
+                        Html = @<text>
+                                   <span class="yearTextForFiltering">@organisation.SectorType.GetAccountingStartDate(report.ReportingYear).ToString("dd/MM/yyyy")</span>
+                                </text>,
+                        Classes = "govuk-!-font-weight-bold"
+                    },
+                    new TableCellViewModel
+                    {
+                        Html = @<text>
+                                   <div>
+                                       @report.GetRequiredToReportOrNotText()
+                                   </div>
+                                   @if (report.CanChangeScope())
+                                   {
+                                       <div>
+                                           <a href="@Url.Action("ChangeOrganisationScope", "Scope", new {encryptedOrganisationId, reportingYear = report.ReportingYear})" class="govuk-link">
+                                               Think this is wrong?<span class="govuk-visually-hidden"> scope for year @report.GetFormattedYearText()</span>
+                                           </a>
+                                       </div>
+                                   }
+                                </text>
+                    },
+                    new TableCellViewModel
+                    {
+                        Html = @<text>
+                                   @{ await Html.RenderPartialAsync("_ReportStatusBadge",
+                                          new ReportStatusBadgeViewModel
+                                          {
+                                              Desktop = true,
+                                              DateText = report.GetReportingDeadline().ToString("d MMMM yyyy"),
+                                              ReportStatus = report.GetReportBadge()
+                                          }); 
+                                   }
+                                </text>
+                    },
+                    new TableCellViewModel
+                    {
+                        Html = @<text>
+                                   <a loadtest-id="create-report-@report.ReportingYear" class="govuk-link"
+                                      href="@Url.Action("ReportOverview", "ReportOverview", new {encryptedOrganisationId, reportingYear = report.ReportingYear, canTriggerLateSubmissionWarning = true})">
+                                       @report.GetReportLinkText()
+                                   </a>
+                                </text>
+                    },
+                }
+            };
+            return newRow;
+        }
+        TableRowViewModel GenerateMobileRow(AllOrganisationReportsForYearViewModel report)

```

Comment:
I'd quite like there to be an extra line here.

Reply:
Agreed and added.